rust/ommx/src/tutorial/artifacts.rs

```diff
@@ -0,0 +1,250 @@
+//! # OMMX Artifacts
+//!
+//! This tutorial demonstrates how to work with OMMX artifacts using the Rust API.
+//!
+//! ## What are OMMX Artifacts?
+//!
+//! OMMX artifacts are OCI-compliant container images that store OMMX messages.
+//! They provide a standardized way to save, share, and reuse optimization problems
+//! and other OMMX messages.
+//!
+//! ## Creating and Saving Artifacts
+//!
+//! You can create and save OMMX artifacts using the `Builder`:
+//!
+//! ```rust,no_run
+//! use ommx::artifact::Builder;
+//! use ommx::v1::{Linear, Instance};
+//! use std::path::Path;
+//! use ocipkg::ImageName;
+//!
+//! // Create a linear function
+//! let linear = Linear::single_term(1, 1.0) + Linear::single_term(2, 2.0) + 3.0;
+//!
+//! // Create an artifact builder
+//! let image_name = ImageName::parse("localhost:5000/linear:latest").unwrap();
+//! let mut builder = Builder::new(image_name).unwrap();
+//!
+//! // Add the linear function to the artifact
+//! let mut instance = Instance::default();
+//! let mut function = ommx::v1::Function::default();
+//! function.function = Some(ommx::v1::function::Function::Linear(linear));
+//! instance.objective = Some(function);
+//! builder.add_instance(instance, ommx::artifact::InstanceAnnotations::default()).unwrap();
+//!
+//! // Save the artifact to a file
+//! let path = Path::new("linear_artifact.oci");
```

Comment:
```suggestion
//! let path = Path::new("linear_artifact.ommx");
```

OMMX Artifactをファイルとして保存する時は `*.oci` ではなく `*.ommx` というファイル名にしてください。他の部分についても同様です。